doc/run-mock-sds-endpoint.md

```diff
@@ -4,20 +4,25 @@ In order to test loading supplementary data, we have a development script that c
 returns mocked supplementary data and mock dataset metadata for that supplementary data.
 
 Ensure the following env var is set before running the script:
+
 ```bash
-SDS_API_BASE_URL=http://localhost:5003/v1/unit_data
+SDS_API_BASE_URL=http://localhost:5003
 ```
 
 From the home directory, run using
+
 ```python
-python -m scripts.mock_sds_endpoint
+python - m
+scripts.mock_sds_endpoint
```

Comment:
this change looks accidental

Reply:
Will be good to get this in, so happy for you to make this change. We'll get a 2nd ✅ from Yuyu.

Reply:
done - and fixed language for the snippet

Reply:
Looks good, happy to resolve.